sys/crypto/Makefile

```diff
@@ -1 +1,7 @@
+ifeq (, ${RIOT_CHACHA_PRNG_DEFAULT})
+    RIOT_CHACHA_PRNG_DEFAULT := $(shell jot -w '0x%08xull,' -r 16 0 0x7ffffffe)
```

Comment:
BTW: you could also check whether ```jot``` exists in the path. If it doesn't exist you can just set this variable to a default value. For testing purposes this should be fine.
However, this needs to be mentioned somewhere (Printing a warning message during compiling should suffice)